test/Microsoft.Azure.Devices.Edge.Test/helpers/ManualProvisioningFixture.cs

```diff
@@ -7,6 +7,92 @@ namespace Microsoft.Azure.Devices.Edge.Test.Helpers
     using Microsoft.Azure.Devices.Edge.Test.Common;
     using NUnit.Framework;
 
+    public class X509ManualProvisioningFixture : ManualProvisioningFixture
+    {
+        public X509Thumbprint thumbprint { get; set; }
+        public EdgeDevice device { get; set; }
+
+        public X509ManualProvisioningFixture()
+            :base()
+        {
+        }
+
+        [OneTimeSetUp]
+        public async Task X509ProvisionEdgeAsync()
+        {
+            await Profiler.Run(
+                async () =>
+                {
+                    using (var cts = new CancellationTokenSource(Context.Current.SetupTimeout))
+                    {
+                        CancellationToken token = cts.Token;
+                        DateTime startTime = DateTime.Now;
+
+                        this.thumbprint = this.CreateSelfSignedCertificateThumbprint();
+
+                        EdgeDevice device = await EdgeDevice.GetOrCreateIdentityAsync(
+                        Context.Current.DeviceId + "-x509",
+                        this.iotHub,
+                        AuthenticationType.SelfSigned,
+                        thumbprint,
+                        token);
+
+                        Context.Current.DeleteList.TryAdd(device.Id, device);
+
+                        this.device = device;
+
+                        await base.ManuallyProvisionEdgeAsync(device, startTime, token);
+                    }
+                },
+            "Completed edge manual provisioning");
+
+        }
+
+        private X509Thumbprint CreateSelfSignedCertificateThumbprint()
+        {
+            return new X509Thumbprint()
+            {
+                PrimaryThumbprint = "9991572f0a02bdc7c89fc032b95d79aca18ef7a3",
+                SecondaryThumbprint = "9991572f0a02bdc7c89fc032b95d79aca18ef7a4"
+            };
+        }
+    }
+
+    public class SasManualProvisioningFixture : ManualProvisioningFixture
+    {
+        public SasManualProvisioningFixture()
+            : base()
+        {
+        }
+
+        [OneTimeSetUp]
+        public async Task SasProvisionEdgeAsync()
+        {
+            await Profiler.Run(
+                async () =>
+                {
+                    using (var cts = new CancellationTokenSource(Context.Current.SetupTimeout))
+                    {
+                        CancellationToken token = cts.Token;
+                        DateTime startTime = DateTime.Now;
+
+                        EdgeDevice device = await EdgeDevice.GetOrCreateIdentityAsync(
+                        Context.Current.DeviceId,
+                        this.iotHub,
+                        AuthenticationType.Sas,
+                        null,
+                        token);
```

Comment:
Indent

Reply:
I'm missing what's wrong - the checkinbuild complains if I change this...